src/pyannote/audio/pipelines/premium/speaker_diarization.py

```diff
@@ -71,6 +71,7 @@ def __call__(
         min_speakers: Optional[int] = None,
         max_speakers: Optional[int] = None,
         model: str = "precision-2",
+        exclusive_diarization: bool = False,
```

Comment:
Shouldn't it be `exclusive` instead?

Reply:
4.0 uses `exclusive_diarization` so I wanted it to be consistent.
But maybe I should instead make 4.0 use `exclusive` also?